articles/hosted-pages/login/lock-passwordless.md

```diff
@@ -0,0 +1,21 @@
+---
+description: How to use Lock Passwordless with the Hosted Login Page
+---
+# Hosted Login with Lock Passwordless
+
+Using Lock Passwordless in the Hosted Login Page is the easiest and most secure implementation of passwordless authentication that can be achieved. Additionally, the Hosted Login Page is the _only_ way to perform passwordless authentication with native applications at this time.
+
+## Lock Passwordless Template for the Hosted Login Page
```

Comment:
I don't see the reason why this title is that long. Simplify. Set it to `Lock Passwordless Template`. The doc is for HLP, no reason to repeat it.